test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -257,32 +257,36 @@ std::string RandomCodeBase::generate(int _maxOpNumber, RandomCodeOptions const&
 		eth::Instruction inst = (eth::Instruction) opcode;
 		eth::InstructionInfo info = eth::instructionInfo(inst);
 
-		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+		while (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
 			|| std::find(invalidOpcodes.begin(), invalidOpcodes.end(), inst) != invalidOpcodes.end())
 		{
 			if (_options.useUndefinedOpCodes)
+			{
 				code += toCompactHex(opcode, 1);
+				return "0x" + code;
+			}
 			else
 			{
 				//Byte code is yet not implemented. do not count it.
-				i--;
+				opcode++;
+				inst = (eth::Instruction) opcode;
+				info = eth::instructionInfo(inst);
 				continue;
```

Comment:
Why is `code` changed in all other cases but not here?

Reply:
random codes are genereated in for sycle.
if a random code is undefined then we change it to a defined opcode in a while loop untill it is defined. (if we dont accept undefined opcodes)

there was an issue with static seed. a random undefined opcode would lead to endless loop.

Reply:
I suggest separating this `findingDefinedOpcode` feature as a function.